python/rainbow/simulators/proximal_contact/jacobi_solver.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import numba as nb
+from rainbow.simulators.proximal_contact.solver_interface import SolverInterface
+
+
+@nb.njit(parallel=True, nogil=True, cache=True)
+def sweep_worker(K, w, z, b, mu, x, friction_solver, hybrid):
+    """ Parallel Jacobi Proximal Algorithm.
+
+    Args:
+        K (int): Contact points number.
+        w (ArrayLike): The WJT.dot(x).
+        z (ArrayLike): The current contact force.
+        b (ArrayLike): b = Ju^t + ∆t JM^{-1}h + EJu^t
+        mu (float): The coefficient of friction.
+        x (ArrayLike): The current contact force.
+        friction_solver (callable): The proximal operator of friction cone function.
+        hybrid (bool): If True, use the hybrid parallel Jacobi algorithm.
+
+    Returns:
+        ArrayLike: The new contact force.
+    """
```

Comment:
This is not the same style of doc comments we use elsewhere. Please be style-consistent.